packages/devtools_shared/lib/src/devtools_api.dart

```diff
@@ -14,6 +14,8 @@ const apiResetDevTools = '${apiPrefix}resetDevTools';
 const apiGetDevToolsFirstRun = '${apiPrefix}getDevToolsFirstRun';
 const apiGetDevToolsEnabled = '${apiPrefix}getDevToolsEnabled';
 const apiSetDevToolsEnabled = '${apiPrefix}setDevToolsEnabled';
+const apiGetConsentMessage = '${apiPrefix}getConsentMessage';
+const apiSetConsentMessageAsShown = '${apiPrefix}setConsentMessageAsShown';
```

Comment:
nit: maybe separate these and add a new header `/// package:unified_analytics properties APIs`